public/app/features/explore/spec/helper/setup.tsx

```diff
@@ -170,20 +224,19 @@ function makeDatasourceSetup({ name = 'loki', id = 1 }: { name?: string; id?: nu
         },
       },
       name: name,
-      uid: `${name}-uid`,
+      uid: uid,
       query: jest.fn(),
-      getRef: jest.fn().mockReturnValue({ type: 'logs', uid: `${name}-uid` }),
+      getRef: () => ({ type, uid }),
       meta,
     } as any,
   };
 }
 
-export const waitForExplore = async (exploreId: ExploreId = ExploreId.left, multi = false) => {
-  if (multi) {
-    return await withinExplore(exploreId).findAllByText(/Editor/i);
-  } else {
-    return await withinExplore(exploreId).findByText(/Editor/i);
-  }
+export const waitForExplore = (exploreId: ExploreId = ExploreId.left, multi = false) => {
```

Comment:
Can we remove `multi` param now? (Also from all usages)

Reply:
good catch! updated it in 5e28f63909